### ISSUE TITLE
Failed to parse trade confirmation because of empty date 

### ISSUE THREAD

**Issue by bolshakov** [KonishchevDmitry/investments#80]
Hey 👋 

The `investments tax-statement` command fails with the following error:

```
E: Error while reading "~/Documents/Interactive Brokers/Statements": Error while reading "~/Documents/Interactive Brokers/Statements/Trade_Confirmation_20220103_20221230.csv": Failed to parse (...) (28 line) record: Invalid date: "".
```

I omitted the specific values in the error message, but I noticed that compared to line 27, the OrderTime and SettleDate values are empty. Let me know if you need more info 🙇

**Comment by KonishchevDmitry** [comment]
Is this line for trade that hasn't been settled yet?

Yes - I need a line from broker statement for this trade and lines for this symbol from trade confirmation statement to understand what is this and how to match one to another. You can change trade volume and other numbers and send it to konishchev@gmail.com for privacy reasons.

**Comment by bolshakov** [comment]
@KonishchevDmitry I sent you an email with the above details. It seems the issue is related to Viatris Inc. spin off from Pfizer Viatris Inc. Pfizer formed a new entity (Viatris Inc.) and distributed its shares among NYSE: PFE stakeholders. Therefore, I got VTRS shares without buying them.

**Comment by KonishchevDmitry** [comment]
Fixed in v5.4.1.

You actually have the following technical trade records in the statement:
* Sell
* Cancellation of the sell
* Sell

which looks like (if you don't try to deal with trade codes):
* Sell
* Buy
* Sell

I haven't tried to collapse them yet (which looks possible). Don't know actually which is better in terms of potential communication with tax inspector.
